web/gui/dashboard_info.js

```diff
@@ -3045,6 +3045,26 @@ netdataDashboard.context = {
     // ------------------------------------------------------------------------
     // eBPF
 
+    'ebpf.tcp_functions': {
+        info: 'TCP functions shows the number of calls for functions \'tcp_sendmsg\', \'tcp_cleanup_rbuf\' and \'tcp_close\' independent of the return to be success or fail.'
```

Comment:
i suggest to use `"text"` to avoid `\`

Reply:
I agree with you that this will remove the `\`, but possibly eBPF will be the unique to have `info` using double quotes, and I did not want to break this pattern.

Reply:
>  and I did not want to break this pattern.

What is the problem?

how about `` then

Reply:
Thank you for the suggestion, I am using `<code>` now.